.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Description
```

Comment:
Probably, we can get more useful info in PRs and reduce time to review and ask questions.
If it works well, can do similar for Issues as well